NineChronicles.Headless.Tests/GraphTypes/States/Models/RankingInfoTypeTest.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nekoyume.Model.State;
+using NineChronicles.Headless.GraphTypes.States;
+using Xunit;
+using static NineChronicles.Headless.Tests.GraphQLTestUtils;
+
+namespace NineChronicles.Headless.Tests.GraphTypes.States.Models
+{
+    public class RankingInfoTypeTest
+    {
+        [Theory]
+        [MemberData(nameof(Members))]
+        public async Task QueryRankingInfo(RankingInfo rankingInfo, object expected)
```

Comment:
```suggestion
        public async Task Query(RankingInfo rankingInfo, object expected)
```

Isn't it enough with `Query` as a test name, like you used in `ArenaInfoTypeTest`?